packages/react-composites/tests/browser/call/hermetic/TeamsClosedCaptions.test.ts

```diff
@@ -0,0 +1,146 @@
+    if (initialState?.call) {
+      initialState.isTeamsCall = true;
+      initialState.call.captionsFeature = {
+        ...captionsFeatureState,
+        isCaptionsFeatureActive: false,
+        startCaptionsInProgress: true
+      };
+    }
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { newControlBarExperience: 'true' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page)).toMatchSnapshot('teams-captions-loading-banner.png');
+  });
+
+  test('Show closed captions banner when enabled', async ({ page, serverUrl }) => {
+    const initialState = defaultMockCallAdapterState();
+    if (initialState?.call) {
+      initialState.isTeamsCall = true;
+      initialState.call.captionsFeature = captionsFeatureState;
+    }
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { newControlBarExperience: 'true' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page)).toMatchSnapshot('show-teams-captions.png');
+  });
+
+  test('Show RTL languages from right to left', async ({ page, serverUrl }) => {
+    const initialState = defaultMockCallAdapterState();
+    if (initialState?.call) {
+      initialState.isTeamsCall = true;
+      initialState.call.captionsFeature = captionsFeatureStateArabic;
+    }
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { newControlBarExperience: 'true' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page)).toMatchSnapshot('show-teams-captions-RTL.png');
+  });
+
+  test('Captions menu shows correct when clicked on desktop', async ({ page, serverUrl }, testInfo) => {
+    if (isTestProfileMobile(testInfo)) {
+      test.skip();
+    }
+    const initialState = defaultMockCallAdapterState();
+    if (initialState?.call) {
+      initialState.isTeamsCall = true;
+      initialState.call.captionsFeature = captionsFeatureState;
+    }
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { newControlBarExperience: 'true' }));
+    await pageClick(page, dataUiId('captions-banner-more-button'));
+    expect(await stableScreenshot(page)).toMatchSnapshot('show-teams-captions-menu-on-banner.png');
+  });
+
+  test('Hide closed captions banner when disabled', async ({ page, serverUrl }) => {
+    const initialState = defaultMockCallAdapterState();
+    if (initialState?.call) {
+      initialState.isTeamsCall = true;
+      initialState.call.captionsFeature = {
+        ...captionsFeatureState,
+        isCaptionsFeatureActive: false,
+        startCaptionsInProgress: false
+      };
+    }
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { newControlBarExperience: 'true' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page)).toMatchSnapshot('hide-teams-captions-banner.png');
```

Comment:
What is the "banner" that is hidden here? (Screenshot is just the regular call page, I don't see the equivalent `show-teams-captions-banner`). Is "banner" the container we show the captions in? If so can just name this something like `'teams-captions-closed'` or `teams-captions-off` (i.e. to avoid ambiguation, match the name of the snapshot with the state instead of the action that was performed to get to that state)

Reply:
This test is for we should not see the captions banner showing up when those two params are false  isCaptionsFeatureActive: false, startCaptionsInProgress: false.
Captions Banner is the container that shows all the captions. 
But yeah the name is a bit confusing, Teams-captions-closed sound good